### PR TITLE
@damassi => [SearchResults] Remove individual entity tabs in favor of one common one

### PR DESCRIPTION
**Pull request by mzikherman**
The Artworks and Artists tab remain their own components. The artworks tab is the filter, and the artists tab has some custom fragments to fetch further artist details.

The remaining 7(!) tabs were thus identical. The only difference is the url fragment/tab route they correspond to, and the 'entities' searched (these are types that may appear on each tab).

So those are consolidated into one component (which does the correct pagination, URL, zero state + feedback form, etc.).

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 9ceb10efc0bcbd6c076233644a532449e676e832

https://deploy-preview-2229--artsy-reaction.netlify.com

File: src/Apps/Search/routes.tsx [hunk 1]
```diff
@@ -1,22 +1,64 @@
...
+  galleries: ["GALLERY"],
+  categories: ["GENE"],
+  articles: ["ARTICLE"],
+  auctions: ["SALE"],
+  more: ["TAG", "CITY", "FAIR", "FEATURE", "INSTITUTION"],
+}
+
+const entityTabs = []
+
+Object.entries(tabsToEntitiesMap).forEach(([key, entities]) => {
```
Comment (damassi) [inline src/Apps/Search/routes.tsx]:
```js
const entityTabs = Object.entries(tabsToEntitiesMap).map(([key, entities]) => { ... })
```

File: src/Apps/Search/routes.tsx [hunk 2]
```diff
@@ -1,22 +1,64 @@
...
+  entityTabs.push({
+    path: key,
+    Component: SearchResultsEntityRoute,
+    render: ({ props, Component }) => {
+      return render({ props, Component }, { entities, tab: key })
+    },
```
Comment (damassi) [inline src/Apps/Search/routes.tsx]:
```js
return render({ 
  props: {
    ...props,
    entities,
    tab: key
  }
})
```
and then the second param in the upper `render` function  can be removed.

File: src/Apps/Search/routes.tsx (hunk 1 above)
Comment (mzikherman) [reply src/Apps/Search/routes.tsx]: oo nice! ⛳️

**Comment by artsyit** [comment]
:rocket: PR was released in v15.11.10 :rocket:

**Comment by mzikherman** [comment]
I was in a 🔥 mood so creeped this with a commit to remove the defunct 'Comparables' app, which has been supplanted by price summaries stuff.

**Comment by damassi** [comment]
awesome 👍 tyy

**Pull request merged**